### DISCUSSION TITLE
Made into vscode plug-in, will be sought after by many people.

### DISCUSSION THREAD

**Discussion by verydemo** [AykutSarac/jsoncrack.com#130] [Ideas]
Looking forward to.

**Comment by AykutSarac** [comment]
Can you elaborate?

**Comment by mg6** [comment]
Would be great if it was possible to run it directly in VS Code, similarly to [draw.io integration](https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio).

**Comment by AykutSarac** [comment]
A VS Code extension is being considered upon requests received here and on Twitter, though other tasks are currently prioritized as [PWA](https://github.com/AykutSarac/jsonvisio.com/releases/tag/v1.6.0) just has been released recently.

**Comment by AndrewLaganaro** [comment]
I support this, this project is simply beautiful, it would be in great use if we can run it in vscode too

**Comment by AykutSarac** [comment]
VS Code extension is being considered for next major release (v2.0.0).

**Comment by noobyogi0010** [comment]
Hey @AykutSarac 
I would like to work on this request if you can guide me through it.
Thanks!

**Comment by worksofliam** [comment]
This would be so neat

**Comment by salgiza** [comment]
A VS Code extension that can integrate with others (e.g., Thunder Client) so that you can easily open the JSON returned by a Thunder Client request, and analyze it with JsonVisio: that would be amazing.

**Comment by AykutSarac** [comment]
If anyone experienced developing VS Code extension we could collaborate, unfortunatelt that's not coming with v2.0

**Comment by noobyogi0010** [comment]
I would love to implement this feature. I think it will be precious to have it on VS Code.
I'm not an expert in VS code extensions but I can try it if someone can guide me a bit.

**Comment by LukasPoque** [comment]
I managed to get the graph rendered as a VSCode extension with live updates, similar to the built-in markdown preview. But to release this I need more understanding of JsonCrack because sometimes the rendering is a little bit of and some things are broken.

If @AykutSarac is okay with it I could copy my code into a separate public repository explicitly for the extension. Then everybody can help to improve it.

Currently, the source code of the graph view is just copied from this repo. I think it would be nicer if we could use a separate package directly so we don't have duplicate code. (see: #42)

**Reply by AykutSarac** [reply]
I'd love to collaborate on it and do an official release. Can you mail to me at the mail address shown at my profile?

**Reply by LukasPoque** [reply]
Yep 👍

**Reply by KarlHeitmann** [reply]
I'd like to collaborate, I have already experimented with VS code extensions (on the extension [git graph](https://github.com/mhutchie/vscode-git-graph/issues/543#issuecomment-890415637), my favorite VS Code extension), so I understand how to develop vs code extensions. Also about a month ago I was reading the `src/utils/json-editor-parser/` folder.

**Comment by Mosaab-Emam** [comment]
I forked this project and will try to make a VS Code extension from it, as well as a desktop app using Tauri.
Wish me luck.

**Reply by AndrewLaganaro** [reply]
Hero

**Comment by KarlHeitmann** [comment]
This is a great idea. I give classes in a bootcamp, maybe if I publish the list of requirements or the project in our slack channel, some students can come and collaborate in simple tasks.

**Comment by AykutSarac** [comment]
Hello guys! VS Code extension is currently in progress, you may test it out here! https://github.com/AykutSarac/jsoncrack-vscode
Looking forward to see your contributions!

**Reply by AndrewLaganaro** [reply]
I've tried to install but Vs code says it is not compatible with my Vs code version 1.70.0
